Formula/edbrowse.rb

```diff
@@ -0,0 +1,84 @@
+class Edbrowse < Formula
+  desc "Command-line editor and web browser"
+  homepage "https://www.edbrowse.org"
+  url "https://github.com/CMB/edbrowse/archive/v3.8.2.tar.gz"
+  sha256 "fed3d2a2ffa5f296ed5edeb5e08ac4cf0f4aec55c8c50945e3f45e300dd3706e"
+  license all_of: ["GPL-2.0-only", "MIT"]
+  head "https://github.com/CMB/edbrowse.git", branch: "master"
+
+  depends_on "make" => :build
+  depends_on "perl" => :build
```

Comment:
Can we use the version provided by macOS?

Reply:
See my comment about make.

Reply:
Use `uses_from_macos`.